source/common/unified_memory_allocation/include/uma/memory_provider_ops.h

```diff
@@ -49,7 +49,7 @@ struct uma_memory_provider_ops_t {
                                            size_t *pageSize);
     enum uma_result_t (*purge_lazy)(void *provider, void *ptr, size_t size);
     enum uma_result_t (*purge_force)(void *provider, void *ptr, size_t size);
-    void (*get_name)(void *provider, const char **ppName);
+    const char *(*get_name)(void *provider);
```

Comment:
nit: looks like a change unrelated to this PR.

Reply:
yes, that's why I've put it into a separate commit, it's just a small improvement